### PR TITLE
Make /challenge work in PMs

### PR DESCRIPTION
**Pull request by urkerab**
As suggested halfway down [this forum post](http://www.smogon.com/forums/posts/7649603).

### CODE REVIEW

**Comment by Zarel** [comment]
You should support passing a format name.

**Comment by urkerab** [comment]
There aren't any PM commands that accept arguments yet, so I had to improvise...

File: js/client-mainmenu.js [hunk 1]
```diff
@@ -314,21 +314,30 @@
 			var $chat = $pmWindow.find('.inner');
 			// this.tabComplete.reset();
 			this.chatHistories[userid].push(text);
-			if (text.toLowerCase() === '/ignore') {
+			var cmd = text;
```
Comment (Zarel) [inline js/client-mainmenu.js]: `= text.toLowerCase()` would save you some repetitive code.

Comment (urkerab) [reply js/client-mainmenu.js]: Well, it's not code that I had to repeat, but I see what you mean.

**Pull request merged**